.github/workflows/clean-aws-resources.yml

```diff
@@ -15,29 +15,47 @@ jobs:
       id-token: write
       contents: read
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-go@v3
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v4
 
       - name: Configure AWS Credentials
-        uses: aws-actions/configure-aws-credentials@v1
+        uses: aws-actions/configure-aws-credentials@v2
         with:
           role-to-assume: ${{ secrets.TERRAFORM_AWS_ASSUME_ROLE }}
           aws-region: us-west-2
 
       - name: Clean old ami
         run: go run ./tool/clean/clean_ami/clean_ami.go --tags=clean
+
+  clean-old-file-systems:
```

Comment:
You can delete this as a result of this PR https://github.com/aws/amazon-cloudwatch-agent/pull/720